docs/040_pepr-tutorials/030_create-pepr-operator.md

```diff
@@ -285,19 +265,20 @@ export async function validator(req: PeprValidateRequest<WebApp>) {
 }
 ```
 
-In this section we generated the CRD class for WebApp, created a function to add `ownerReferences` to the manifests that will be deployed by the Operator to handle deletion of Kubernetes objects, registered the CRD, and added a validator to validate that instances of WebApp are in valid namespaces.
+In this section we generated the CRD class for WebApp, created a function to auto register the CRD, and added a validator to validate that instances of WebApp are in valid namespaces and have a maximum of 7 replicas.
 
 ## Create Helpers
 
-In this section we will create helper functions to help with the reconciliation process.
+In this section we will create helper functions to help with the reconciliation process. The idea is that this operator will "remedy" any accidental deletions of the resources it creates. If any object deployed by the Operator is deleted for any reason, the Operator will abruptly redeploy the object.
 
-Create a `controller` folder in the `capabilities` folder and create a `generators.ts` file in the `capabilities` folder. This file will contain the functions that will generate the manifests that will be deployed by the Operator with the ownerReferences added to them.
+Create a `controller` folder in the `capabilities` folder and create a `generators.ts` file. This file will contain the functions that will generate the manifests that will be deployed by the Operator with the ownerReferences added to them. Since these resources are owned by the WebApp resource, they will be deleted when the WebApp resource is deleted.
```

Comment:
```suggestion
Create a `controller` folder in the `capabilities` folder and create a `generators.ts` file. This file will contain the functions that will generate the Kubernetes Objects that will be deployed by the Operator with the ownerReferences added to them. Since these resources are owned by the WebApp resource, they will be deleted when the WebApp resource is deleted.
```